config.go

```diff
@@ -47,6 +47,8 @@ type Config struct {
 	LogOutput io.Writer
 	// HTTPClient is the http client
 	HTTPClient *http.Client
+	// wait time (in milliseconds) between repetitive requests to the API during polling
+	PollingWaitTime int
```

Comment:
Would type this as `time.Duration` to maximize flexibility for users and be consistent with `marathonClient`.